src/caretogether-pwa/src/Families/FamilyScreen.tsx

```diff
@@ -123,7 +124,10 @@ export function FamilyScreen() {
 				return true;
 			} else if (opt === ArrangementFilterOptionLabel.Cancelled && arrangement.cancelledAtUtc !== undefined) {
 				return true;
-			}
+			} else if (opt === ArrangementFilterOptionLabel.Ended && arrangement.cancelledAtUtc !== undefined) {
```

Comment:
```suggestion
			} else if (opt === ArrangementFilterOptionLabel.Ended && arrangement.endedAtUtc !== undefined) {
```
I think the original code you wrote has a logic bug in it? Take a look at this proposed edit and see if that's the case. Either way, when you know it's correct, include a screenshot or two in your PR showing the correct logic with some sample data that you've entered to demonstrate that you've tested it.